personal_access_tokens.go

```diff
@@ -57,7 +57,14 @@ func (p PersonalAccessToken) String() string {
 // https://docs.gitlab.com/ee/api/personal_access_tokens.html#list-personal-access-tokens
 type ListPersonalAccessTokensOptions struct {
 	ListOptions
-	UserID *int `url:"user_id,omitempty" json:"user_id,omitempty"`
+	CreatedAfter   *time.Time `url:"created_after,omitempty" json:"created_after,omitempty"`
+	CreatedBefore  *time.Time `url:"created_before,omitempty" json:"created_before,omitempty"`
+	LastUsedAfter  *time.Time `url:"last_used_after,omitempty" json:"last_used_after,omitempty"`
+	LastUsedBefore *time.Time `url:"last_used_before,omitempty" json:"last_used_before,omitempty"`
```

Comment:
I guess these `*time.Time` fields should be `*ISOTime` looking at the docs. Can you check/confirm?

Reply:
I check gitlab api document and confirm that you are right，we need to use *ISOTime instead of *time.Time
I have change the type